options/rtdl/generic/linker.cpp

```diff
@@ -279,6 +281,11 @@ void ObjectRepository::_fetchFromPhdrs(SharedObject *object, void *phdr_pointer,
 			object->tlsAlignment = phdr->p_align;
 			object->tlsImageSize = phdr->p_filesz;
 			tls_offset = phdr->p_vaddr;
+		case PT_INTERP:
+			object->interpreterPath = frg::string<MemoryAllocator>{
+				(char*)(phdr->p_vaddr),
+					getAllocator()
+			};
```

Comment:
Do we need to respect `->p_filesz` or is this string always null-terminated according to the ELF spec?

Reply:
It's null terminated.